help/direct-messages.md

```diff
@@ -19,14 +19,19 @@ and do not appear in your list of streams.
 
 {tab|desktop-web}
 
-1. Click the **New direct message** button at the bottom of the app, or
-   use the <kbd>X</kbd> keyboard shortcut.
+1. Click the **New direct message** button at the bottom of the app or the
+   **Start new conversation** button if you are in a direct message view.
```

Comment:
Tiny nit: I think we generally use an Oxford comma before the `or`.

Reply:
Applies elsewhere in this PR as well.

Reply:
We do, good catch!